README.md

```diff
@@ -1,2 +1,213 @@
+The `wso2/msonedrive` module contains operations for accessing the items stored in OneDrive.
+
+## Compatibility
+|                     |    Version     |
+|:-------------------:|:--------------:|
+| Ballerina Language  | 1.2.0   |
+| Microsoftgraph REST API | v1.0          |
+
+## Getting started
+
+1.  Refer the [Getting Started](https://ballerina.io/learn/getting-started/) guide to download and install Ballerina.
+
+2.  To use the Microsoft Graph API, you need to provide the following configuration information in the ballerina.conf file:
+
+       - MS_CLIENT_ID
+       - MS_CLIENT_SECRET
+       - MS_ACCESS_TOKEN
+       - MS_REFRESH_TOKEN
+       - TRUST_STORE_PATH
+       - TRUST_STORE_PASSWORD
+       - WORK_BOOK_NAME
+       - WORK_SHEET_NAME
+       - TABLE_NAME
+
+    Following steps should be followed to obtain the above mentioned configuration information.
+
+    Before you run the following steps you may have to create an account in [OneDrive](https://onedrive.live.com). Next, sign into [Azure Portal - App Registrations](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade). You may have to use your personal or work or school account.
+
+    From the App registrations page, click on New registration. Enter a meaningful name in the name field. 
+    In the Supported account types section, select Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com). Click Register to create the application.
+
+    Copy the Application (client) ID (\<MS_CLIENT_ID>). This is the unique identifier for your app.
+    In the application's list of pages (under the Manage tab in left hand side menu), select Authentication.
+    Under the Platform configurations click on "Add a platform" button.
+    Under the "Configure platforms", click on "Web" button located under the Web applications.
+
+    Under the redirect URIs text box, put [OAuth2 Native Client](https://login.microsoftonline.com/common/oauth2/nativeclient).
+    Under the Implicit grant select Access tokens.
+    Click on Configure.
+    Under the Certificates & Secrets, create a new client secret (\<MS_CLIENT_SECRET>). This requires providing a description and a period of expiry. Next, click on Add button.
+
+    Next, we need to obtain an access token and a refresh token to invoke the Microsoft Graph API.
+    First, in a new browser enter this URL by replacing the \<MS_CLIENT_ID> with the application ID.
+
+    ```
+    https://login.microsoftonline.com/common/oauth2/v2.0/authorize?response_type=code&client_id=<MS_CLIENT_ID>&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient&scope=Files.ReadWrite openid User.Read Mail.Send Mail.ReadWrite offline_access
+    ```
+
+    This may prompt you to enter the username and password for signing into  Azure Portal App.
```

Comment:
```suggestion
    This will prompt you to enter the username and password for signing into the Azure Portal App.
```
May sounds you are unsure about the operation. Therefore, better to use "will" or directly say "This prompts..".

Reply:
Ok, updated accordingly